templates/frontend/objects/chapter.tpl

```diff
@@ -47,6 +47,12 @@
 				datePublished=$publication->getData('datePublished')|date_format:$dateFormatShort
 				urlRecentVersion=$latestVersionUrl|escape
 			}
+			{if $currentContext->getData('enableDois') == true && $currentContext->getData('doiVersioning') == false}
+				<br>
+				{translate key="submission.outdatedVersion.doiDescription"
```

Comment:
I cannot find this locale key...

Reply:
With the latest changes, I've since removed this message/locale key.